### ISSUE TITLE
App crashes after closing and opening new sub-window

### ISSUE THREAD

**Issue by abldabl** [Kingtous/rustdesk_desktop_multi_window#17]
OS: Linux - Ubuntu 22.04.2 LTS

Steps to reproduce:
1) open first sub-window
2) close first sub-window
3) open second sub-window

app crashes without logic. it can crash after 2 sub-window creations or after 3 or 4 or 5.

**Comment by Kingtous** [comment]
Yes, it's expected unfortunately... You can cache your window, not to close but hide the window and hold the windowId.

**Comment by Kingtous** [comment]
Note that Its not the bug for this plugin, it's the drawback of multiple flutter engine shared with one process. The engine is not designed for multiple engine on the head.

**Comment by abldabl** [comment]
i got you, thanks

only one question:
i couldn't run app with your library on windows 10

error was about: 
couldn't find window_size_plugin.h
did you know something? or i should give full error?

**Comment by Kingtous** [comment]
> i got you, thanks
> 
> only one question:
> i couldn't run app with your library on windows 10
> 
> error was about: 
> couldn't find window_size_plugin.h
> did you know something? or i should give full error?
> 

i have added some required header of plugins for rustdesk, u can clean them if your flutter project doesn't contain these plugins.

**Comment by Kingtous** [comment]
> > i got you, thanks
> > 
> > only one question:
> > i couldn't run app with your library on windows 10
> > 
> > error was about: 
> > couldn't find window_size_plugin.h
> > did you know something? or i should give full error?
> > 
> 
> i have added some required headers of plugins for rustdesk, u can clean them if your flutter project doesn't contain these plugins.
